readme_release.md

```diff
@@ -0,0 +1,34 @@
+# HOWTO Release navitia
+
+You should first have a proper python env:
+```
+pip install -r requirements_release.txt -U
+```
+
+Then for a "Normal" release (minor):
+```
+cd <path/to/repo/navitia>
+# to be sure to launch the correct version of the release script
+git fetch <canaltp_distant_repo_name> && git rebase <canaltp_distant_repo_name>/dev dev
```

Comment:
I can change it to `merge` or `checkout <canaltp_distant_repo_name>/dev` no pb
That's just that I always rebase, but not important, the checkout of distant is the minimum.